pkg/config/config.go

```diff
@@ -77,7 +81,11 @@ func Load(path string, netMode netmode.Magic, relativePath ...string) (Config, e
 // be updated based on the provided relative path.
 func LoadFile(configPath string, relativePath ...string) (Config, error) {
```

Comment:
We need a test for it. I'm not sure if it's possible, need to check: try to remove/rename the `config` directory in the start of the test (this change should be reverted in Cleanup) and then run the `LoadFile` with default config path and privnet config.